runtime/runtime/src/verifier.rs

```diff
@@ -477,6 +481,28 @@ fn validate_deploy_contract_action(
     Ok(())
 }
 
+/// Validates `DeployGlobalContractAction`. Checks that the given contract size doesn't exceed the limit.
+fn validate_deploy_global_contract_action(
+    limit_config: &LimitConfig,
+    action: &DeployGlobalContractAction,
+    current_protocol_version: ProtocolVersion,
+) -> Result<(), ActionsValidationError> {
+    if !checked_feature!("stable", GlobalContracts, current_protocol_version) {
+        return Err(ActionsValidationError::UnsupportedProtocolFeature {
+            protocol_feature: "GlobalContracts".to_owned(),
+            version: current_protocol_version,
+        });
+    }
+    if action.code.len() as u64 > limit_config.max_contract_size {
```

Comment:
Confirming, is this the only check we need in the verifier? I'm guessing it's based off the above `validate_deploy_contract_action` function?

Reply:
yes, I've copied this from verifier for the regular deploy contract action, I believe we should enforce the same limits for the global contract code